types/meteor/meteor.d.ts

```diff
@@ -64,15 +106,32 @@ declare module "meteor/meteor" {
 
         /** Method **/
         interface MethodThisType {
+            /** Access inside a method invocation. Boolean value, true if this invocation is a stub. */
             isSimulation: boolean;
+            /** The id of the user that made this method call, or `null` if no user was logged in. */
             userId: string | null;
+            /** Access inside a method invocation. The connection that this method was received on. `null` if the method is not associated with a connection, eg. a server initiated method call. Calls to methods made from a server method which was in turn initiated from the client share the same `connection`. */
             connection: Connection | null;
+            /**
+             * Set the logged in user.
+             * @param userId The value that should be returned by `userId` on this connection.
+             */
             setUserId(userId: string): void;
```

Comment:
You've corrected the `ddp.d.ts` file to allow `null`s. I guess applies here as well?